### PR TITLE
Add indexing statistics

### LINKED ISSUE #3072: Change indexing statistics implementation
TODO: Clearly define new scope

### PR DESCRIPTION
**Pull request by imsdu**
Fixes #3072

+ Get rid of `akka-cluster`/`akka-distributed-data`

### CODE REVIEW

File: delta/app/src/main/resources/akka.conf [hunk 1]
```diff
@@ -4,7 +4,7 @@ akka {
 
   http {
     server {
-      parsing.max-content-length = ${akka.remote.artery.advanced.maximum-frame-size}
+      parsing.max-content-length = 256 KiB
```
Comment (imsdu) [inline delta/app/src/main/resources/akka.conf:7]: This is the value that was previously referenced to

File: delta/plugins/blazegraph/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/routes/BlazegraphViewsRoutesSpec.scala [hunk 2]
```diff
@@ -219,6 +203,39 @@ class BlazegraphViewsRoutesSpec
       }
     }
 
+    "fail to fetch statistics and offset from view without resources/read permission" in {
```
Comment (imsdu) [inline delta/plugins/blazegraph/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/routes/BlazegraphViewsRoutesSpec.scala:206]: I moved the tests up as the update on the view introduce a restriction on the tag

File: tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/SparqlViewsSpec.scala [hunk 3]
```diff
@@ -185,15 +185,15 @@ class SparqlViewsSpec extends BaseSpec with EitherValuable with CirceEq {
       }
     }
 
-    "fetch statistics for defaultSparqlIndex" ignore eventually {
+    "fetch statistics for defaultSparqlIndex" in eventually {
```
Comment (olivergrabinski) [inline tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/SparqlViewsSpec.scala:188]: There is a failed event in this test

File: tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/ElasticSearchViewsSpec.scala [hunk 4]
```diff
@@ -223,7 +223,7 @@ class ElasticSearchViewsSpec extends BaseSpec with EitherValuable with CirceEq {
       }
     }
 
-    "fetch statistics for testView" ignore eventually {
+    "fetch statistics for testView" in eventually {
```
Comment (olivergrabinski) [inline tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/ElasticSearchViewsSpec.scala:226]: The expected result might need to be updated?

File: tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/SparqlViewsSpec.scala (hunk 3 above)
Comment (imsdu) [reply tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/SparqlViewsSpec.scala:188]: Yes, the failed one is related to the bug I am fixing now

**Pull request merged**